TMS_Tests/Tests/AddRemoveElementsTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using OpenQA.Selenium;
+
+namespace TMS_Tests.Tests
+{
+    internal class AddRemoveElementsTest : BaseTest
+    {
+        [Test]
+        public void AddRemoveElements()
+        {
+            Driver.Navigate().GoToUrl("http://the-internet.herokuapp.com/add_remove_elements/");
+            Driver.FindElement(By.XPath("//button[text()='Add Element']")).Click();
+            Driver.FindElement(By.XPath("//button[text()='Add Element']")).Click();
+            Driver.FindElement(By.XPath("//button[text()='Delete']")).Click();
+            var deleteButtons = Driver.FindElements(By.XPath("//button[text()='Add Element']"));
```

Comment:
You are trying to find Add Element buttons "//button[text()='Add Element']" not delete. Need to change locator

Reply:
Updated